bin/theano-cache

```diff
@@ -12,25 +13,25 @@ _logger = logging.getLogger('theano.bin.theano-cache')
 
 def print_help(exit_status):
     if exit_status:
-        print 'command "%s" not recognized' % (' '.join(sys.argv))
-    print 'Type "theano-cache" to print the cache location'
-    print 'Type "theano-cache help" to print this help'
-    print 'Type "theano-cache clear" to erase the cache'
-    print 'Type "theano-cache list" to print the cache content'
-    print 'Type "theano-cache unlock" to unlock the cache directory'
-    print 'Type "theano-cache cleanup" to delete keys in the old format/code version'
-    print 'Type "theano-cache purge" to force deletion of the cache directory'
-    print ('Type "theano-cache basecompiledir" '
-            'to print the parent of the cache directory')
-    print ('Type "theano-cache basecompiledir list" '
-            'to print the content of the base compile dir')
-    print ('Type "theano-cache basecompiledir purge" '
-            'to remove everything in the base compile dir, '
-            'that is, erase ALL cache directories')
+        print('command "%s" not recognized' % (' '.join(sys.argv)))
+    print('Type "theano-cache" to print the cache location')
+    print('Type "theano-cache help" to print this help')
+    print('Type "theano-cache clear" to erase the cache')
+    print('Type "theano-cache list" to print the cache content')
+    print('Type "theano-cache unlock" to unlock the cache directory')
+    print('Type "theano-cache cleanup" to delete keys in the old format/code version')
+    print('Type "theano-cache purge" to force deletion of the cache directory')
+    print('Type "theano-cache basecompiledir" ')
```

Comment:
Parentheses were already used here for string concatenation, and it seems `2to3` got confused by that.